bin/k8s-deploy

```diff
@@ -152,6 +152,24 @@ do
     echo "Updating ${DEPLOYMENT_FILE}-${CI_SHA1}"
     DEPLOYMENT_FILE=${DEPLOYMENT_FILE}-${CI_SHA1}
   fi
+  if [[ $( grep -i replicas $DEPLOYMENT_FILE | awk -F: {'print $2'} | grep -i hpa ) ]]
```

Comment:
The reason this code exists likely won't be obvious in the future. Please add some comments explaining why it is needed and a link to the upstream issue for Kubernetes.